tests/baselines/reference/moduleAugmentationDeclarationEmit1.js

```diff
@@ -0,0 +1,68 @@
+//// [tests/cases/compiler/moduleAugmentationDeclarationEmit1.ts] ////
+
+//// [map.ts]
+
+import { Observable } from "./observable"
+
+(<any>Observable.prototype).map = function() { }
```

Comment:
But you're declaring an extension to `Observable<T>` below, so doesn't that mean `Observable<any>` should have a member named `map` of type `map<U>(proj: (e: any) => U): Observable<U>`?